### PR TITLE
Rework NFT standards source (erc721/1155)

### PR DESCRIPTION
**Pull request by 0xRobin**
### Background

Working on [this PR#2985](https://github.com/duneanalytics/spellbook/pull/2985), one of the issues was duplication in the blur trades due to a join with `nft_transfers` that doesn't always produce a singular result. 
https://github.com/duneanalytics/spellbook/blob/8eeb2ec447b590488e589ea3b0217851c2ee15e5/models/blur/ethereum/blur_ethereum_events.sql#L100-L110

this join only serves the purpose of determining the correct NFT standard (erc721/1155). As seen here:
https://github.com/duneanalytics/spellbook/blob/8eeb2ec447b590488e589ea3b0217851c2ee15e5/models/blur/ethereum/blur_ethereum_events.sql#L25
To get rid of this join logic, this PR aims to include the NFT standard in the `tokens.nft` tables for all known NFT contract addresses.
While working on this, I also took on the chore of cleaning up up some of the logic and standardizing the setup across chains. 

## Changes:

1. Moved manual nft token lists from `tokens_<chain>_nft` to `tokens_<chain>_nft_curated`
2. Removed `category` column from curated nft token lists, this was not present on all chains and empty in 90%+
3. Removed `standard` column from curated nft token lists, this will be pulled in automatically now which removes risk of manual errors.
4. Added `tokens_<chain>_nft_standards` incremental models which collect the token addresses and erc standards from `nft_<chain>_transfers` (which is based on raw ERC721/1155 evt tables)
5. Re-added the overall `tokens_chain_nft` model. This combines every known NFT contract with the extra name information present in the `nft_curated` models
6. Added tests to check the `standard` column and guarantee uniqueness of `contract_address` in nft token models. 

Small fixes:
1. Fixed erc721 reference in the transfer-id for erc1155 single transfers in `nft_<chain>_transfers` models.
2. Fixed a small bug in `tokens_optimism.nft_bridged_mapping`.

### CODE REVIEW

**Comment by 0xRobin** [comment]
Side note: (to-do: tag optimism peeps)
I've smuggled a fix for `tokens_optimism.nft_bridged_mapping` in this PR due to stricter tests. In the old setup it was possible to introduce duplicate bridged token addresses by depositing with a wrong L2 contract reference. 

To fix this I changed the source from the `BridgeInitialized` evt on L1 to the `BridgeFinalized` evt on L2.
this:
```sql 
select * from optimism_ethereum.L1ERC721Bridge_evt_ERC721BridgeInitiated
where remoteToken = 0x80874cfb4b8ef71be7cb9f9b30be12ac42f7b182
``` 
contains an example duplicate where a wrong `remoteToken` is given by the depositor

the fix:
```sql
select * from ovm_optimism.L2ERC721Bridge_evt_ERC721BridgeFinalized
where localToken = 0x80874cfb4b8ef71be7cb9f9b30be12ac42f7b182
```
shows that only the correct remote/local token combinations make it into the L2 evt table.

File: models/tokens/arbitrum/tokens_arbitrum_nft_standards.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{{ config(
+        alias ='nft_standards',
+        materialized='incremental',
```
[the remark this answers is not in the document]
Comment (jeff-dude) [reply models/tokens/arbitrum/tokens_arbitrum_nft_standards.sql:3]: should apply to all the blockchains

**Comment by jeff-dude** [comment]
> 2\. Removed `category` column from curated nft token lists, this was not present on all chains and empty in 90%+

i'm comfortable with this, but we risk breaking dashboards/queries if users are actually using it?

File: models/tokens/arbitrum/tokens_arbitrum_nft_standards.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (0xRobin) [reply models/tokens/arbitrum/tokens_arbitrum_nft_standards.sql:3]: good catch!

**Pull request merged**